### ISSUE TITLE
Training Configuration Inquiry

### ISSUE THREAD

**Issue by YWCandGHY** [boschresearch/DiffVLA#5]
The experimental you mentioned in the paper:
For **DiffVLA**:

- Total batch size: 256
- Learning rate: 1e-4
- Epochs: 100

For **DiffVLA++**:

- Total batch size: 8
- Learning rate: 1e-4
- Epochs: 30

How should the experimental configuration be set in the publicly available code version?

**Comment by yugao2011** [comment]
sorry for the miss-leading, please follow

Total batch size: 8
Learning rate: 1e-4
Epochs: 30 

The EPDM scorer needs a very large trajectory vocabulary (>=8192) to reach a good performance. This cost a lost of VRAM, so we   change to small total batch size on duo to the limited computation resource we have. 
We did not find a optimal way to handle this large VRAM consumption duo to the limited time of the competition.

**Comment by YWCandGHY** [comment]
> sorry for the miss-leading, please follow
> 
> Total batch size: 8 Learning rate: 1e-4 Epochs: 30
> 
> The EPDM scorer needs a very large trajectory vocabulary (>=8192) to reach a good performance. This cost a lost of VRAM, so we change to small total batch size on duo to the limited computation resource we have. We did not find a optimal way to handle this large VRAM consumption duo to the limited time of the competition.

Thank you for your prompt reply. Have you tried using dynamically computed metric scores to calculate the loss?
